pkg/ratelimit/option.go

```diff
@@ -89,6 +83,6 @@ func InitLimiter() Option {
 			return InAllowList
 		}
 		l.limiters.LoadOrStore(label, newLimiter())
-		return ConcurrencyChanged
+		return LimiterNotChanged
```

Comment:
Has there been a change here, as expected?

Reply:
Yes, the limit has not been changed when we initialize the limiter.